module/VuFind/src/VuFind/Db/Entity/SearchEntityInterface.php

```diff
@@ -40,4 +42,170 @@
  */
 interface SearchEntityInterface extends EntityInterface
 {
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int;
+
+    /**
+     * Get user.
+     *
+     * @return UserEntityInterface
+     */
+    public function getUser(): int;
```

Comment:
The return type was fixed in the comment, but it's out of sync in the actual code.